callstats-java-sdk/src/main/java/io/callstats/sdk/CallStats.java

```diff
@@ -0,0 +1,114 @@
+package io.callstats.sdk;
```

Comment:
Probably need to double check this with @vr000m We might need to add some license stuff at the head of every file. refer  to this for example: https://github.com/alphazero/jredis/blob/master/core/ri/src/main/java/org/jredis/ri/alphazero/JRedisClient.java